### DISCUSSION TITLE
AAVE “Transaction failed” error message

### DISCUSSION THREAD

**Discussion by jaystats2** [WalletConnect/walletconnect-monorepo#3077] [Developer Support]
I can’t redeem my AAVE staking rewards. How do I update my WalletConnect? Thank you.

**Comment by crypblizz8** [comment]
Hey @jaystats2, sorry to hear that. Can you share how you are running into this error?
- Wallet
- Platform of Wallet
- Website you are using
- How are you doing each step?

Cheers

**Comment by jaystats2** [comment]
I’m on the AAVE safety module/staking page.I’m connected to TrustWallet. Please help me fix this.Thanks ; )Sent from my iPhoneOn Jul 10, 2023, at 6:48 AM, crypblizz ***@***.***> wrote:﻿
Hey @jaystats2, sorry to hear that. Can you share how you are running into this error?

Wallet
Platform of Wallet
Website you are using
How are you doing each step?

Cheers

**Comment by BritvaBo** [comment]
How you solve this problem?
